src/main/java/org/kiwiproject/elk/ElkAppenderFactory.java

```diff
@@ -28,24 +29,27 @@
 
     private static final JsonHelper JSON_HELPER = JsonHelper.newDropwizardJsonHelper();
 
-    private String host;
-    private int port;
     private boolean useUdp;
     private boolean includeCallerData;
     private boolean includeContext = true;
     private boolean includeMdc = true;
     private Map<String, String> customFields = new HashMap<>();
     private Map<String, String> fieldNames = new HashMap<>();
 
+    private ElkLoggerConfigProvider elkLoggerConfigProvider = ElkLoggerConfigProvider.builder().build();
```

Comment:
Would we instead want to initialize this in the no-args constructor and blow up if the provider can't provide ?

Reply:
I really thought about making this a static property, but it was harder to test.  Without knowing for sure the error handling being done by DW and/or Logback I don't know if they handle exceptions during construction like they do for build.

Reply:
OK, well, the field initialization happens at instance construction time, i.e. when someone calls the constructor, so I don't think it matters much in this case.